Dockerfile.docs

```diff
@@ -0,0 +1,25 @@
+# NOTE: `mkdocs` only seems to work on either Python 2.7 or 3.6 (despite upstream docs claiming support for other 3.x versions)
+FROM python:3.6-alpine
```

Comment:
@cosmix I'm sorry; I was looking at some netlify errors today and noticed we are inf act using Python 3.6 -- So I re-tested and it works fine. I must have thought I tried 3.6 and ran into bugs too but I am mistaken. mkdocs only seems to work on 2.7 and 3.6 (despite their website and pypi claminig other versions). I will fix/ammend this comment / FROM line :)

Reply:
And... Done.

Reply:
Hold on. They claim comparability with 3.7 too. Can you check it please?

Reply:
I can and will comment with the results.

Reply:
I re-tested a bunch of different Python versions and here are the resutls:

```bash
prologic@Jamess-iMac
Tue Jan 14 07:28:07
~/NetData/netdata
 (mkdocs_docker_image) 0
$ ./test.sh
Building netdata/docs:py3.3 ... ERR
Running netdata/docs:py3.3 ... ERR
Building netdata/docs:py3.4 ... ERR
Running netdata/docs:py3.4 ... ERR
Building netdata/docs:py3.5 ... OK
Running netdata/docs:py3.5 ... OK
Building netdata/docs:py3.6 ... OK
Running netdata/docs:py3.6 ... OK
Building netdata/docs:py3.7 ... OK
Running netdata/docs:py3.7 ... OK
Building netdata/docs:py3.8 ... OK
Running netdata/docs:py3.8 ... OK
```

I can't explain some bugs / Tracebacks I was seeing at the meetup; but *meh* I'll amend with 3.8 and move on :)

Reply:
Script if you want to verify/confirm:

```bash
#!/bin/sh

for PYVER in 3.3 3.4 3.5 3.6 3.7 3.8; do
  printf "Building netdata/docs:py%s ... " "$PYVER"
  if docker build -q --build-arg PYVER="$PYVER" -f Dockerfile.docs -t netdata/docs:py"$PYVER" . > "${PYVER}".build.log 2>&1; then
    printf "OK\n"
  else
    printf "ERR\n"
  fi

  printf "Running netdata/docs:py%s ... " "$PYVER"
  if docker run -i -t --rm -v "$PWD":/netdata netdata/docs:py"$PYVER" > "${PYVER}".log 2>&1; then
    printf "OK\n"
  else
    printf "ERR\n"
  fi
done
```